src/shared/common/forms/mapped-display.jsx

```diff
@@ -36,7 +36,7 @@ class MappedDisplay extends Component {
             return (renderCheckbox(label, this.props.showLabel, choices[0].label));
         } else {
             return (
-                <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value}</span>
+                <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value.replaceAll(',', ', ')}</span>
```

Comment:
Should we be more specific in the else that it's rendering a checkbox-group?

Reply:
Yeah I get what you're saying. It might not be a checkbox group though, it could be any component that isn't specifically a checkbox. I did have a version of this where it called a method to render a checkbox group but the only difference in that method was the addition off the replaceAll so I decided to just combine it here into line 39